khal/ui/__init__.py

```diff
@@ -1061,6 +1061,8 @@ def __init__(self, collection, conf=None, title='', description=''):
             elistbox.set_focus_date_callback = lambda _: None
         self.calendar = ContainerWidget(calendar)
         self.lwidth = 31 if self._conf['locale']['weeknumbers'] == 'right' else 28
+        if self._conf['view']['frame'] == "width" or self._conf['view']['frame'] == "color":
```

Comment:
I think `in` would be easier to read here, and possibly (from very brief testing) faster.
```suggestion
        if self._conf['view']['frame'] in ["width", "color"]:
```